src/lib/isIdentityCard.js

```diff
@@ -130,73 +127,70 @@ const validators = {
     return true;
   },
   'zh-CN': (str) => {
-    const provinceAndCitys = {
-      11: '北京',
-      12: '天津',
-      13: '河北',
-      14: '山西',
-      15: '内蒙古',
-      21: '辽宁',
-      22: '吉林',
-      23: '黑龙江',
-      31: '上海',
-      32: '江苏',
-      33: '浙江',
-      34: '安徽',
-      35: '福建',
-      36: '江西',
-      37: '山东',
-      41: '河南',
-      42: '湖北',
-      43: '湖南',
-      44: '广东',
-      45: '广西',
-      46: '海南',
-      50: '重庆',
-      51: '四川',
-      52: '贵州',
-      53: '云南',
-      54: '西藏',
-      61: '陕西',
-      62: '甘肃',
-      63: '青海',
-      64: '宁夏',
-      65: '新疆',
-      71: '台湾',
-      81: '香港',
-      82: '澳门',
-      91: '国外',
-    };
+    const provincesAndCities = [
```

Comment:
Any reasons for this change?

Reply:
In the original version we are just checking if the first two numbers of the ID are a valid province number. It means having an object is unnecessary and probably the result of a copy/paste

Reply:
Sorry for late response. So that we don't lose the info on the provinces, could you add those as comments on each line instead?
```js
11, // '北京'
...
```

Reply:
Done